.travis.yml

```diff
@@ -18,4 +18,4 @@ install: true
 script:
   - make .gitvalidation
   - make lint
-  - make binaries
+  - make ocid ocic
```

Comment:
Forgot to remember that make integration runs a container so I fixed conmon build only there (cannot fix the Travis instance)